build/npm/v2-jf/package-lock.json

```diff
@@ -1,5 +1,17 @@
 {
   "name": "jfrog-cli-v2-jf",
   "version": "2.48.0",
-  "lockfileVersion": 1
+  "lockfileVersion": 2,
+  "requires": true,
+  "packages": {
+    "": {
+      "name": "jfrog-cli-v2-jf",
+      "version": "2.48.0",
```

Comment:
This will make the release a little bit harder because we will need to change this version every time. It worked perfectly before the change.